packages/frontend/app/PublicEnv.tsx

```diff
@@ -0,0 +1,23 @@
+export interface PublicEnvironment {
+  OPEN_PAYMENTS_URL: string
+}
+
+declare global {
+  interface Window {
+    ENV: PublicEnvironment
+  }
+}
+
+type PublicEnvProps = {
+  env: PublicEnvironment
+}
+
+export const PublicEnv = ({ env }: PublicEnvProps) => {
+  return (
+    <script
+      dangerouslySetInnerHTML={{
+        __html: `window.ENV = ${JSON.stringify(env)}`
+      }}
+    />
+  )
+}
```

Comment:
Is the `OPEN_PAYMENTS_URL` actually getting used anywhere? I see that we're making it available in the browser but it seems like its unused?